packages/app-store/apps.keys-schemas.generated.ts

```diff
@@ -55,6 +56,7 @@ export const appKeysSchemas = {
   make: make_zod_ts,
   matomo: matomo_zod_ts,
   metapixel: metapixel_zod_ts,
+  "mock-payment-app": mock_payment_app_zod_ts,
```

Comment:
I don't think we should have this in the actual list of apps in the app store. We need to put this in a place that's specific to the E2E tests

Reply:
Addressed